Cargo.toml

```diff
@@ -1,8 +1,12 @@
 [package]
 name = "vmm-sys-util"
 version = "0.1.0"
-authors = ["Jing Liu <jing2.liu@linux.intel.com>"]
-license = "Apache-2.0"
+authors = ["Intel Virtualization Team <vmm-maintainers@intel.com>"]
+description = "A system utility set for virtualization"
+repository = "https://github.com/rust-vmm/vmm-sys-util"
+readme = "README.md"
+keywords = ["vmm"]
```

Comment:
I don't think that vmm is the best keyword here because this is more of a "utils" crate and it's not directly related to vmms.